config/webpacker.yml

```diff
@@ -3,60 +3,96 @@
 default: &default
   source_path: app/javascript
   source_entry_path: packs
+  public_root_path: public
   public_output_path: packs
   cache_path: tmp/cache/webpacker
+  webpack_compile_output: true
 
   # Additional paths webpack should lookup modules
   # ['app/assets', 'engine/foo/app/assets']
-  additional_paths: ['app/javascript/src']
+  additional_paths: ['app/javascript/src'] # Allows using absolute imports in ts files w/o 'src/'
 
-  # Reload manifest.json on all requests so we reload latest compiled packs
+  # Reload manifest.json on all requests so we reload latest compiled packs
   cache_manifest: false
 
+  # Extract and emit a css file
+  extract_css: false
+
+  static_assets_extensions:
+    - .jpg
+    - .jpeg
+    - .png
+    - .gif
+    - .tiff
+    - .ico
+    - .svg
+    - .eot
+    - .otf
+    - .ttf
+    - .woff
+    - .woff2
+
   extensions:
-    - .coffee
-    - .erb
-    - .js
-    - .jsx
-    - .ts
     - .tsx
-    - .vue
+    - .ts
+    - .jsx
+    - .mjs
+    - .js
     - .sass
     - .scss
     - .css
+    - .module.sass
+    - .module.scss
+    - .module.css
     - .png
     - .svg
     - .gif
     - .jpeg
     - .jpg
-    - .yml
     - .yaml
 
 development:
   <<: *default
   compile: true
-  webpack_compile_output: true
-  check_yarn_integrity: false
 
+  # Reference: https://webpack.js.org/configuration/dev-server/
   dev_server:
+    https: false
     host: localhost
     port: 3035
+    public: localhost:3035
     hmr: false
-    https: false
+    # Inline should be set to true if using HMR
+    inline: true
+    overlay: true
+    compress: true
+    disable_host_check: true
+    use_local_ip: false
+    quiet: false
+    pretty: false
+    headers:
+      'Access-Control-Allow-Origin': '*'
+    watch_options:
+      ignored: '**/node_modules/**'
+
 
 test:
   <<: *default
   compile: false
   webpack_compile_output: true
 
+
   # Compile test packs to a separate directory
   public_output_path: packs-test
 
 production:
   <<: *default
 
   # Production depends on precompilation of packs prior to booting for performance.
-  compile: false
+  compile: true
```

Comment:
Why change this? I think it was correct before, we precompile on the image build and deliver the assets inside the image.